RealEstate/RealEstate/Pages/Index.cshtml

```diff
@@ -6,90 +6,91 @@
     List<HouseData.Houses> houses = (List<HouseData.Houses>)ViewData["Houses"];
     List<LandData.Lands> lands = (List<LandData.Lands>)ViewData["Lands"];
 }
+<body style="background-color: #d6c3c3">
+    <div class="text-center">
+        <h1 class="display-4" style="background-color: #d0a4a4">Welcome to Real Estate</h1>
+        <!-- <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p> -->
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.